api/EventTarget.json

```
@@ -310,10 +310,10 @@
                   "version_added": "≤18"
                 },
                 "firefox": {
-                  "version_added": "50"
+                  "version_added": "49"
```

OK, I did some more digging and this definitely didn't ship in 49. See https://bugzilla.mozilla.org/show_bug.cgi?id=1287706.

```suggestion
                  "version_added": "50"
```